services/QuillLMS/engines/evidence/app/models/evidence/research/gen_ai/example_prompt_response_feedback.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: evidence_research_gen_ai_example_prompt_response_feedbacks
+#
+#  id                         :bigint           not null, primary key
+#  evaluation                 :text
+#  feedback                   :text             not null
+#  label                      :string           not null
+#  created_at                 :datetime         not null
+#  updated_at                 :datetime         not null
+#  passage_prompt_response_id :integer          not null
+#
+module Evidence
+  module Research
+    module GenAI
+      class ExamplePromptResponseFeedback < ApplicationRecord
+        belongs_to :passage_prompt_response, class_name: 'Evidence::Research::GenAI::PassagePromptResponse'
+
+        validates :feedback, presence: true
```

Comment:
Tiny take or leave, but naming-wise, you might consider something like `text` for this field to make calls `feedback.text` vs. `feedback.feedback`